crypto/bn/asm/rsaz-3k-avx512.pl

```diff
@@ -48,8 +48,17 @@
     $avx512ifma = ($1==2.11 && $2>=8) + ($1>=2.12);
 }
 
-if (!$avx512 && `$ENV{CC} -v 2>&1` =~ /((?:clang|LLVM) version|.*based on LLVM) ([0-9]+\.[0-9]+)/) {
-    $avx512ifma = ($2>=7.0);
+if (!$avx512 && `$ENV{CC} -v 2>&1`
```

Comment:
Unless this script is included by something else (which I couldn't find) `$avx512` is always going to be falsy here (and also at line 46) - is that intentional?

Reply:
I noticed that too (preexisting problem in the script, not caused by this PR). I assume `$avx512` was intended to be `$avx512ifma`?

Reply:
IMO not a big deal, but I can add additional fix to the PR for this - this is on master branch too